packages/atomic/src/components/common/checkbox.tsx

```diff
@@ -54,6 +54,7 @@ export const Checkbox: FunctionalComponent<CheckboxProps> = (props) => {
       onMouseDown={(e) => props.onMouseDown?.(e)}
     >
       <atomic-icon
+        style={{stroke: 'white'}}
```

Comment:
Using a inline style gives (0, 1, 0) priority and using a part selector gives (0, 1, 1) priority.

Reply:
This is quite bizarre because it works in atomic testing and in the storybook. When I specify a new stroke in the part, it overrides it visually but in the browser dev tools it is crossed out. I am looking into it to understand better.

```
atomic-rating-facet::part(value-checkbox-icon) {
        stroke: aqua;
        background-color: red;
      }
```
      

<img width="446" alt="Screenshot 2023-05-16 at 1 29 47 PM" src="https://github.com/coveo/ui-kit/assets/132079077/ecd289cd-ba66-40ee-b5b2-13afefa9ed2b">

<img width="370" alt="Screenshot 2023-05-16 at 1 29 59 PM" src="https://github.com/coveo/ui-kit/assets/132079077/a34d39fd-492d-47f5-87c7-ff9c94a33499">

Reply:
As you said the browser is doing pretty weird 💩 here.
However, it 'works' so far, so... :shipit: